src/main/java/atmosfears/AtmosFearsBE/service/AirParticulatesService.java

```diff
@@ -0,0 +1,21 @@
+package atmosfears.AtmosFearsBE.service;
+
+import atmosfears.AtmosFearsBE.database.AirParticulatesRepository;
+import atmosfears.AtmosFearsBE.model.AirParticulates;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Service
+public class AirParticulatesService {
+
+    @Autowired
```

Comment:
```suggestion
```
`@Autowired` is not needed in here as well. The `@AllArgsConstructor` from `lombok` creates all constructors and one of them is a constructor that takes the `AirParticulatesRepository` as a parameter. The `constructor` injection will be performed automatically, thanks to the constructor created by `@AllArgsConstructor` so we don't need to inject this repository manually.